wasmbinding/test/custom_message_test.go

```diff
@@ -194,6 +196,52 @@ func (suite *CustomMessengerTestSuite) TestCreateDenomMsg() {
 	suite.Equal([][]byte(nil), data)
 }
 
+func (suite *CustomMessengerTestSuite) TestSetDenomMetadataMsg() {
+	codeID := suite.StoreTestCode(suite.ctx, suite.contractOwner, "../testdata/reflect.wasm")
```

Comment:
It looks like we do not use the contract in any testcase in the package.
Lets either remove contrract related code or use wasmKeeper.Execute instead of  suite.executeCustomMsg/messenger.Dispatch